packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx

```diff
@@ -107,6 +107,7 @@ function TenantSelector() {
           if (tenant) {
             toast.success(t('tenants.tenant_created', { name: tenant.name }));
             void mutate();
+            window.location.assign(new URL(`/${tenant.id}`, window.location.origin).toString());
```

Comment:
Have you checked if the newly selected tenant is the current one? If yes, then we don't need to navigate the page